media/BUILD.gn

```diff
@@ -113,6 +113,12 @@ rtc_static_library("rtc_media_base") {
     "base/videocommon.h",
     "base/videosourcebase.cc",
     "base/videosourcebase.h",
+
+    #d3d11 frame source files, move these where it makes sense
+    "base/d3d11_frame_buffer.h",
+    "base/d3d11_frame_buffer.cc",
+    "base/d3d11_frame_source.cc",
+    "base/d3d11_frame_source.h",
```

Comment:
Should be moved to `media\base`.

Reply:
They're already there, or did you mean some other place?

Reply:
I suggested `media/base` because source base classes are implemented there. But it could be left in `base`  or `base/win` for the same reason as above.